src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,241 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+
+import java.util.Scanner;
+
+public class Duke {
+    private static final String LS = System.lineSeparator();
```

Comment:
Just curious if the name _LS_ would be confusing as NEWLINE seems to be more intuitive.